### ISSUE TITLE
Add date picker to GFW map in ranges

### ISSUE THREAD

**Issue by ygoex** [National-Forestry-Authority/forests#289]

**Comment by peacog** [comment]
This is ready for testing @LeScruf 
See for example https://forests.stg.envs.utils.nfa.go.ug/asset/799/gfw

**Comment by LeScruf** [comment]
Hi @peacog  - this seems to be still a little bit of work in progress?
1. The date picker probably needs an additional time frame which could be 'previous 4 weeks' (a rolling time interval) and/or a whole month picker (so that variations in alerts from month to month can be compared / followed) i.e. the user can select Feb and see Feb's alerts, or January to see that month's alerts. The reason being is that at the beginning of a month, the time interval selectable is very limited, so we need to provide more options (without crashing the view with a data overload). I admit I am shifting the goal posts - but the early month limitation had not occurred to me alas.
2. Some CFRs seem to be strangely missing - such as Lalak, Lamwo and Agoro Agu from the link provided above. Not sure if this is a wider system problem?

**Comment by peacog** [comment]
@LeScruf I've opened a ticket to investigate the problem of the missing CFRs. See #317

**Comment by peacog** [comment]
Hi @LeScruf 
I updated our code to use the latest version of the GFW apis because the old version we were using is no longer available. After doing that and a bit of refactoring the alerts load very quickly. I've reverted back to the default date picker because there isn't a performance problem any more when loading the last 3 months of data. Take a look at Achwa River range: https://forests.stg.envs.utils.nfa.go.ug/asset/799/gfw

We will revisit #274 to see if we can load the alerts at country level efficiently.

**Comment by LeScruf** [comment]
@peacog Yes - MUCH faster - looking good. I have created a new issue related to this - [#337](https://github.com/National-Forestry-Authority/forests/issues/337) - don't worry it's a small and quick - but useful - UX improvement. Thanks.
